gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -19,13 +19,13 @@
 import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
Unrelated to the PR but if you can clean the redundant "com.google.gcloud.storage." in line 1532 (update javadoc) that would be great.